File: ProjectFinalAdminBD/src/java/dominio/Venta.java
package dominio;

/**
 * @author emfam
 */
public class Venta {
    private int idAutomovil;
    private int cedula;
    private String usuario;
    private double precioVenta;
    
    public Venta(){}

    public Venta(int idAutomovil, int cedula, String usuario, double precioVenta) {
        this.idAutomovil = idAutomovil;
        this.cedula = cedula;
        this.usuario = usuario;
        this.precioVenta = precioVenta;
    }

    public int getIdAutomovil() {
        return idAutomovil;
    }

    public void setIdAutomovil(int idAutomovil) {
        this.idAutomovil = idAutomovil;
    }

    public int getCedula() {
        return cedula;
    }

    public void setCedula(int cedula) {
        this.cedula = cedula;
    }

    public String getUsuario() {
        return usuario;
    }

    public void setUsuario(String usuario) {
        this.usuario = usuario;
    }

    public double getPrecioVenta() {
        return precioVenta;
    }

    public void setPrecioVenta(double precioVenta) {
        this.precioVenta = precioVenta;
    }

    @Override
    public String toString() {
        return "Venta{" + "idAutomovil=" + idAutomovil + ", cedula=" + cedula + ", usuario=" + usuario + ", precioVenta=" + precioVenta + '}';
    }
    
}

File: ProyectoBD2021/domain/Venta.php
<?php
	
	class Venta
	{

		private $idAutomovil;
		private $cedula;
		private $usuario;
		private $precioTotal;

		function __construct($idAutomovil,$cedula,$usuario,$precioTotal)
		{
			$this->idAutomovil = $idAutomovil;
			$this->cedula = $cedula;
			$this->usuario = $usuario;
			$this->precioTotal = $precioTotal;
		}

		public function getIdVenta()
		{
			return $this->idVenta;
		}

		public function getVenta()
		{
			return $this->venta;
		}

	}
 ?>